server/routes/webmaker-app.js

```diff
@@ -16,11 +17,27 @@ var webmakerVersion = require('../../package.json').dependencies.webmaker;
 if (!habitat.get('FIREBASE_ENDPOINT')) throw new Error('You must configure FIREBASE_ENDPOINT in .env');
 
 var firebase = new Firebase(habitat.get('FIREBASE_ENDPOINT') + '/apps');
+var ircBot = habitat.get('BOT_ENDPOINT');
```

Comment:
if you need to have this in a variable, i would go with constant-style `var BOT_ENDPOINT = habitat.get('BOT_ENDPOINT')`